README.md

```diff
@@ -1241,23 +1244,28 @@ and the Dfinity project. [\[18\]][18]
 The Lightning Network is a proposed message relay network operating at a layer
 above the Bitcoin blockchain, enabling many orders of magnitude improvement in
 transaction throughput by moving the majority of transactions outside of the
-consensus ledger into so-called "payment channels". This is made possible (with
-great difficulty) by the Bitcoin scripting language, which enables parties to
+consensus ledger into so-called "payment channels".
+<!---
+lightning network exists beyond bitcoin
+--->
+Which enables parties to
 enter into stateful contracts where the state can be updated by sharing digital
 signatures, and contracts can be closed by finally publishing evidence onto the
 blockchain, a mechanism first popularized by cross-chain atomic swaps.  By
 openning payment channels with many parties, participants in the Lightning
-Network can become focal points for routing the payments of others, leading to a
-fully connected payment channel network, at the cost of capital being tied up on
-payment channels.
-
-While the Lightning Network can also easily extend across multiple independent
-blockchains to allow for the transfer of _value_ via an exchange market, it
-cannot be used to assymetrically transfer _tokens_ from one blockchain to
-another.  The main benefit of the GnuClear network described here is to enable
-such direct token transfers.  That said, we expect payment channels and the
-Lightning Network to become widely adopted along with our token transfer
-mechanism, for cost-saving and privacy reasons.
+Network can become focal points for routing the payments, and contracts of others, leading to a
+fully connected channel network, at the cost of capital being tied up on
+payment channels. This cost can be completely removed by making it into
```

Comment:
I said something confusing things before. sorry.

of course the interest cost exists.
How many bitcoin would you pay right now to get 10 bitcoin in a month?
10 bitcoin minus that number is the monthly interest rate for bitcoin.

> If we can find some use to those bonded reserve-pool collateral tokens, we'll implement it.

That is the exact opposite of what I am suggesting. you are currently reusing the bonds for 2 purposes. I am telling you that you can't reuse bonds that way.

> I'm not convinced that channels are the solution

Neither am I. You don't need channels to solve this problem. That is just how I happened to solve it for flying fox. DPOS could solve this problem for example.

>Adding complexity here will likely skew the incentives and make the system insecure.
You are being complicated by reusing bonds. I am trying to convince you to simplify it. use bonds for only security against double-spend. Don't reuse them for deciding who has validating power.

If you reuse bonds for both protecting against double-signing, and for deciding who has validating power, it will do both things poorly.
It will be too many bonds for stopping double-signing, so you will waste money by the interest rate.
It will be too few bonds for deciding who should have consensus, so you will have retirement attacks.